app/models/concerns/api_authenticable.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ApiAuthenticable
+  AUTHENTICATION_TOKEN_LENGTH = 64
+
+  extend ActiveSupport::Concern
+
+  included do
+    before_create do
+      self.authentication_token = generate_authentication_token
+    end
+
```

Comment:
Do we always want to create an authentication_token for the users?
How about just supplying the method to create it, but let it up to the controllers to actually create it when needed?

Reply:
I believe then it would have a high risk of being overlooked. 
It's fairly easy to remove it if it's not being done. I believe we are going to have this discussion repeated times in the future due to the structure of this template.

Reply:
I agree with Hari here, this is easy to turn off. I would assume it is needed in most cases.